kong/init.lua

```diff
@@ -97,8 +96,8 @@ local function load_plugins(kong_conf, dao)
 
   -- load installed plugins
   for plugin in pairs(kong_conf.plugins) do
-    if plugin == "galileo" then
-      ngx_log(ngx.WARN, "the 'galileo' plugin has been deprecated")
+    if constants.DEPRECATED_PLUGINS[plugin] then
+      ngx.log(ngx.WARN, "the '" .. plugin .. "' plugin has been deprecated")
```

Comment:
ditto: as per the above comment, no Lua-land concatenation and we should use the same message